tests/cpu/Context_tests.cpp

```diff
@@ -119,8 +119,8 @@ OCIO_ADD_TEST(Context, use_searchpaths)
     const std::string res1 = searchPath1 + "/Context.cpp";
     OCIO_CHECK_ASSERT(strcmp(SanitizePath(resolvedSource.c_str()).c_str(),
                              SanitizePath(res1.c_str()).c_str()) == 0);
-    OCIO_CHECK_NO_THROW(resolvedSource = context->resolveFileLocation("GPUHelpers.h"));
-    const std::string res2 = searchPath2 + "/GPUHelpers.h";
+    OCIO_CHECK_NO_THROW(resolvedSource = context->resolveFileLocation("GPUUnitTest.h"));
+    const std::string res2 = searchPath2 + "/GPUUniTest.h";
```

Comment:
You're missing a "t".  "GPUUnitTest.h".   Please ensure your tests are passing locally before you push.

Reply:
I only ran the gpu tests. Sorry about that. Fixed now.

I'm getting two failures in tests I did not change in the CPU tests suite. Is that normal?

Reply:
No, all the CPU tests should pass (they do on the CI machines). You could try to fix them or log them as issues.